src/commands/registries/azure/deployImageToAci.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { IActionContext, parseError } from 'vscode-azureextensionui';
+import { ext } from '../../../extensionVariables';
+import { localize } from '../../../localize';
+import { ContextTreeItem } from '../../../tree/contexts/ContextTreeItem';
+import { registryExpectedContextValues } from '../../../tree/registries/registryContextValues';
+import { RemoteTagTreeItem } from '../../../tree/registries/RemoteTagTreeItem';
+import { executeAsTask } from '../../../utils/executeAsTask';
+import { execAsync } from '../../../utils/spawnAsync';
+import { addImageTaggingTelemetry } from '../../images/tagImage';
+
+export async function deployImageToAci(context: IActionContext, node?: RemoteTagTreeItem): Promise<void> {
+    if (!node) {
+        node = await ext.registriesTree.showTreeItemPicker<RemoteTagTreeItem>([registryExpectedContextValues.dockerHub.tag, registryExpectedContextValues.dockerV2.tag], context);
+    }
+
+    const aciContext = await ext.contextsTree.showTreeItemPicker<ContextTreeItem>([/aciContext;/i], context);
+
+    // Switch to the other context if needed
+    if (!aciContext.current) {
+        await vscode.commands.executeCommand('vscode-docker.contexts.use', aciContext);

```

Comment:
Since we specify the ACI context explicitly for `docker run`, this is not strictly necessary. I assume this is done so that the user gets visual feedback that their deployment action succeeded.

Reply:
Exactly. @BigMorty asked specifically to have it switch to the ACI context. Technically the `--context` is no longer necessary on the `docker run` command, but, since we're doing things in multiple contexts already I thought it was safe to just leave it there.